src/Functions/array/arrayResize.h

```diff
@@ -16,13 +16,12 @@ class FunctionArrayResize : public IFunction
 
     bool isVariadic() const override { return true; }
     size_t getNumberOfArguments() const override { return 0; }
-
+    bool useDefaultImplementationForConstants() const override { return true; }
+    bool useDefaultImplementationForNulls() const override { return false; }
     bool isSuitableForShortCircuitArgumentsExecution(const DataTypesWithConstInfo & /*arguments*/) const override { return true; }
 
     DataTypePtr getReturnTypeImpl(const DataTypes & arguments) const override;
     ColumnPtr executeImpl(const ColumnsWithTypeAndName & arguments, const DataTypePtr & return_type, size_t input_rows_count) const override;
-    bool useDefaultImplementationForConstants() const override { return true; }
```

Comment:
@rschu1ze Why remove these? Just curious.

Reply:
I just moved that up to l. 19/20.

These functions are "trait functions", they describe properties of the function and it is nicer to keep them all in one place.